CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.0.2 (2021-09-07)
```

Comment:
PRs should not bump this. This is up to maintainer to decide what version bump to make.